.buildkite/commands/dependency-tree-diff.sh

```diff
@@ -7,18 +7,20 @@ DIFF_DEPENDENCIES_FILE="$DIFF_DEPENDENCIES_FOLDER/diff_dependencies.txt"
 CONFIGURATION="wordpressVanillaReleaseRuntimeClasspath"
 DEPENDENCY_TREE_VERSION="1.2.0"
 
-echo "--> Starting the check"
+REPO_HANDLE="wordpress-mobile/wordpress-android"
 
-git config --global user.email '$( git log --format='%ae' $CIRCLE_SHA1^! )'
-git config --global user.name '$( git log --format='%an' $CIRCLE_SHA1^! )'
+COMMIT_HASH=$BUILDKITE_COMMIT
+PR_NUMBER=$BUILDKITE_PULL_REQUEST
+PR_URL="https://api.github.com/repos/$REPO_HANDLE/pulls/$PR_NUMBER"
+CURRENT_BRANCH=$BUILDKITE_BRANCH
 
-currentBranch=$(git rev-parse --abbrev-ref HEAD)
+echo "--> Starting the check"
 
-prNumber="${CIRCLE_PULL_REQUEST##*/}"
-githubUrl="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$prNumber"
+git config --global user.email '$( git log --format='%ae' $COMMIT_HASH^! )'
+git config --global user.name '$( git log --format='%an' $COMMIT_HASH^! )'
 
-echo "--> Fetching the target branch from $githubUrl"
-githubResponse="$(curl "$githubUrl" -H "Authorization: token $GITHUB_API_TOKEN")"
+echo "--> Fetching the target branch from $PR_URL"
+githubResponse="$(curl "$PR_URL" -H "Authorization: token $GITHUB_TOKEN")"
 targetBranch=$(echo "$githubResponse" | tr '\r\n' ' ' | jq '.base.ref' | tr -d '"')
```

Comment:
This whole dance should not be necessary in Buildkite because, unlike CircleCI, Buildkite [already exposes the target branch as an env var](https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_PULL_REQUEST_BASE_BRANCH), which should prevent us from making this now-useless network request, and thus stop requiring us to do the `git config` dance as well.